content/en/docs/releases/next-release-preview/index.md

```diff
@@ -71,3 +71,14 @@ https://github.com/spinnaker/orca/pull/4481
 https://github.com/spinnaker/rosco/pull/998
 
 For more information please see [the README](https://github.com/spinnaker/kork/blob/18d1c6e88597a9147851b37412ea38b3fd7032d5/kork-artifacts/src/main/java/com/netflix/spinnaker/kork/artifacts/README.md).
+
+### Rosco
+https://github.com/spinnaker/rosco/pull/986 adds in support for [Helmfile](https://helmfile.readthedocs.io/) as a bake manifest templating engine to Rosco. 
+
+For configuration please refer to the [deploy-helm user guide](https://github.com/spinnaker/spinnaker.io/blob/master/content/en/docs/guides/user/kubernetes-v2/deploy-helm/index.md#Helmfile)
```

Comment:
This link will only be live once we merge to master. I am not sure the approach taken by the code owners to self-referential references